modevo-consistency/src/test/java/test4giis/consistency/TestCheckConsistency.java

```diff
@@ -0,0 +1,145 @@
+			String nameTable = iteratorNameTables.next();
+			PreparedStatement ps = buildInsert (nameTest, nameTable, tableColumns.get(nameTable));
+			preparedStatementsTable.put(nameTable, ps);
+		}
+		try {
+			initDB (preparedStatementsTable, keyspace, tableQueryInit);
+		} catch (Exception e) {
+			throw new RuntimeException (e);
+		}
+		testScript (name.getMethodName(), connection);
+		try {
+			compareCassandraSQL(tableQueryCompare, keyspace);
+		} catch (Exception e) {
+			throw new RuntimeException (e);
+		}
+	}
+   
+	/**
+	 * Method to build a PreparedStatement to insert data to all columns of a Cassandra table
+	 */
+	private PreparedStatement buildInsert (String keyspace, String nameTable, List<String> columns) {
+    	StringBuilder insert = new StringBuilder ("INSERT INTO ");
+    	String a = "\""+keyspace+"\"."+nameTable;
+    	System.out.print(a);
+    	insert.append(a).append(" (");
+    	int numberColumns = columns.size();
+    	StringBuilder placeholders = new StringBuilder();
+    	for (int i=0; i<numberColumns;i++) {
+    		insert.append(columns.get(i));
+    		if (i+1 == numberColumns) {
+    			insert.append(")");
+    			placeholders.append("?)");
+    		}
+    		else {
+    			insert.append(", ");
+    			placeholders.append("?, ");
+
+    		}
+    	}
+    	insert.append(" VALUES (").append(placeholders).append(";");
+    	System.out.print(insert.toString());
+    	return connection.getSession().prepare(insert.toString());
+    }
+    /**
+     * Method to start the migration of data from the SQL database to the Cassandra database to populate it before using MoDEvo
+     * @param keyspace 
+     * @param tableQuery 
+     */
+    private void initDB(Map<String, PreparedStatement> preparedStatementsTable, String keyspace, Map<String, String> tableQuery) throws Exception{
+		java.sql.Connection connectionSQL = new MySQLAccess().connect(keyspace);
+		Oracle oc = new Oracle();
+		Set<String> nameTables =  preparedStatementsTable.keySet();
+		Iterator<String> iteratorNameTables = nameTables.iterator();
+		while (iteratorNameTables.hasNext()) {
+			String nameTable = iteratorNameTables.next();
+			oc.sqlQueryMigrate(keyspace, nameTable, tableQuery.get(nameTable), connectionSQL, connection, preparedStatementsTable.get(nameTable));
+		}
+		connectionSQL.close();
+
+	}
+    /**
+     * Method to call the oracle. Right now it includes hardcodded the query for the only experimental subject tested. 
+     * In a following version this will be generalized and obtained from an external source depending on the experimental subject.
+     * @param tableQuery 
+     * @param keyspace 
+     */
+    private void compareCassandraSQL(Map<String, String> tableQuery, String keyspace) throws Exception{
+		MySQLAccess mysql = new MySQLAccess();
+		Connection con = mysql.connect(keyspace);
+		Oracle oc = new Oracle();
+		boolean result = oc.oracleCompare(tableQuery, name.getMethodName(), PROPERTIES, con);
+		con.close();
+		assertTrue (result);
```

Comment:
This is the worst approach to compare two things because it prevents the debugging when the test fails.
Comparison should allow to see whera the differences are:
- All needed file should be published to allow an external comparison of their differences
- Better, use the visual-assert component at my github, that is designed for this. It compares strings (form the files) and produces an html file with the differences.